uniffi/src/ffi/rustbuffer.rs

```diff
@@ -365,7 +365,11 @@ mod test {
         // We don't want to actually materialize a huge vec, so unsafety it is!
         // This won't cause problems because the contained items are Plain Old Data.
```

Comment:
Have these tests ever been run through miri? I'm somewhat certain undef is not okay even for Plain Old Data, and you have to go through `MaybeUninit` instead.

Reply:
I don't think this should block the PR, but `MaybeUninit` does seem better to me.  I'll make a separate PR for that.

Reply:
Ben ended up killing these tests in #1216

Reply:
Yeah I noticed 🙂